README.md

```diff
@@ -1 +1,20 @@
-# Hello-Microverse
\ No newline at end of file
+
+# Hello-Microverse
+## Microverse Greetings Web Page
+
+Built With
+CSS
+HTML
+
+Author: Susana Gonzalez
+GitHub: @Zuz4n
+
+🤝 Contributing
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the issues page.
+
+*_Show your support_*
+Give a ⭐️ if you like this project!
+
+📝 License: MIT licensed.
```

Comment:
- [ ] Good job adding information to your `Readme` file as requested by the previous reviewer 😺, but it should look professional. Kindly, use this [template](https://github.com/microverseinc/readme-template/blob/master/README.md) and adapt it with information related to your project. 🦖